cetmix_tower_server/models/cx_tower_file.py

```diff
@@ -123,6 +123,15 @@ class CxTowerFile(models.Model):
     file = fields.Binary(
         attachment=True,
     )
+    variable_ids = fields.Many2many(
```

Comment:
Why do we need to redefine this field completely if it's already defined in the mixin?
If you want to put relation explicitly you can add just it:

```suggestion
variable_ids = fields.Many2many(
    relation="cx_tower_file_variable_rel",
    column1="file_id",
    column2="variable_id",
)
```